assembl/static2/js/app/pages/debate.jsx

```diff
@@ -18,6 +21,7 @@ class Debate extends React.Component {
     this.showThumbnails = this.showThumbnails.bind(this);
     this.hideThumbnails = this.hideThumbnails.bind(this);
     this.displayThumbnails = this.displayThumbnails.bind(this);
+    this.redirectToV1Threads = this.redirectToV1Threads.bind(this);
```

Comment:
I have no information about expected behavior of other phases than phase 1 and 2.